src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
```

Comment:
The `ColorSupplier` class should not be package-private; it should be public to be accessible from other packages if needed.